src/components/Tab/Tab.scss

```diff
@@ -2,18 +2,132 @@
 
 .nitrozen-tab-container {
   display: flex;
-  width: 100%;
+  height: 4.8rem;
+  position: relative;
+  &.n-tab-scroll {
+    padding: 0 6.4rem;
+    overscroll-behavior-x: none;
+    .nitrozen-tab {
```

Comment:
change all .nitrozen- prefixes to .n- as discussed in the nitrozen-contributors group